benchmark/DMap.hs

```diff
@@ -64,11 +83,11 @@ buildBigMap 1 x = insert (typeRep @a) x
 buildBigMap n x = insert (typeRep @a) x
                 . buildBigMap (n - 1) (Proxy @(a + 1))
 
-rknf :: TypeRepMap f -> ()
-rknf = rnf . map (\(This t) -> typeRepFingerprint t) . keys
+newtype Hack f = Hack (TypeRepMap f)
 
-prepareBenchDMap :: IO ()
-prepareBenchDMap = evaluate (rknf bigMap)
+instance NFData (Hack f) where
+  rnf (Hack x) = 
+    rnf . map (\(This t) -> typeRepFingerprint t) $ keys x
```

Comment:
Yes, it's used in `env` in the appropriate modules. This `TypeRepMap` was defined locally and it's actually `DMap`, so instance was given for the other type. I think it was done so benchmarks can be written with a backpack or copypasted, but it may lead to unexpected results, like now.